src/openforms/forms/models/form_variable.py

```diff
@@ -41,14 +45,17 @@ class FormVariableManager(models.Manager):
     use_in_migrations = True
 
     @transaction.atomic
-    def create_for_form(self, form: "Form") -> None:
+    def create_for_form(self, form: "Form", fix_prefill_config=False) -> None:
+        """create FormVariable objects for `form`. `fix_prefill_config` is used only
+        for patching migrations involving corrupted data; set for removal"""
```

Comment:
https://peps.python.org/pep-0257/ has something to say about docstring formats. No need to fix this now, but in the future please use the following format:

```py
"""
Create FormVariable objects for the given ``form``.

``fix_prefil_config`` is used only for patching migrations involving corrupted data and
is set for removal.
"""
```